django_telegrambot/apps.py

```diff
@@ -43,42 +45,58 @@ class DjangoTelegramBot(AppConfig):
     name = 'django_telegrambot'
     verbose_name = 'Django TelegramBot'
     ready_run = False
-    bot_tokens = []
-    bot_usernames = []
-    dispatchers = []
-    bots = []
-    updaters = []
+    bots_data = OrderedDict()
```

Comment:
why you choose an ordered dict here? most of the time you took just the values and use them as a list. Why not just define bots_data as list of dicts?

Reply:
IDK why i did this 😂 you are completely right 
I had my reason at first but probably the code didn’t go that way to use dict characteristic